src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx

```diff
@@ -227,22 +229,22 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
 
   const { ownedVenues } = useOwnedVenues({});
 
-  const backButtonOptionList = ownedVenues.filter(
-    ({ id, name, template, worldId }) => {
-      if (portal?.worldId !== worldId || id === portalId) {
-        return null;
-      }
-
-      return {
-        name,
-        template,
-      };
-    }
-  );
+  const backButtonOptionList =
```

Comment:
Same comments as `backButtonOptionList` in `SpaceEditForm` both for replacing `reduce()` with `Object.fromEntries()` and `useMemo()`

Reply:
done eef5fce